src/Model/List/PaginatedList.php

```diff
@@ -532,14 +532,6 @@ public function PrevLink()
         }
     }
 
-    /**
-     * Returns the total number of items in the list
-     */
-    public function TotalItems()
-    {
-        return $this->getTotalItems();
-    }
```

Comment:
This method alias isn't really doing any good, seems sensible to remove it.
Removing it from here requires removing it from the superclass as well, so adding the getter version there in its place.